.github/config/evergreen-arm-hardfp-rdk.json

```diff
@@ -0,0 +1,40 @@
+{
+  "docker_service": "build-raspi",
+  "bootloader": "raspi-2",
```

Comment:
I believe you want this to be `rdk`. This field is used in MH to determine which device to run tests on. See https://github.com/youtube/cobalt/actions/runs/6647382573/job/18108519692?pr=1779:
```
--loader_platform raspi-2
```

Reply:
Adding some offline conversation. This flag controls both the build toolchain and the on-device test device type. If it will require an override there's a fair amount of plumbing to get this flag to the on device test trigger.

What is the plan with regards to having an RDK toolchain? Is it possible to setup a toolchain alias or similar that would invoke the raspi toolchain for rdk? Other than pretending that we're building raspi.